### PR TITLE
add left padding to gracenote group

### LINKED ISSUE #1363: OSMD: Grace note colliding
> * [ ]  Grace note colliding in test_clef_end_measure (measure 1)
>   image

### PR DESCRIPTION
**Pull request by rvilarl**
fixes #1363

Visual differences (bravura in order diff, current, reference), notes with grace notes move two pixels to the right.

Accidental Automatic_Accidentals Bravura
Accidental Automatic_Accidentals Bravura_current
Accidental Automatic_Accidentals Bravura_reference
Grace_Notes Grace_Note_Basic Bravura
Grace_Notes Grace_Note_Basic Bravura_current
Grace_Notes Grace_Note_Basic Bravura_reference
Grace_Notes Grace_Note_Basic_with_Slurs Bravura
Grace_Notes Grace_Note_Basic_with_Slurs Bravura_current
Grace_Notes Grace_Note_Basic_with_Slurs Bravura_reference
Grace_Notes Grace_Note_Slash Bravura
Grace_Notes Grace_Note_Slash Bravura_current
Grace_Notes Grace_Note_Slash Bravura_reference
Grace_Notes Grace_Note_Slash_with_Beams Bravura
Grace_Notes Grace_Note_Slash_with_Beams Bravura_current
Grace_Notes Grace_Note_Slash_with_Beams Bravura_reference
Grace_Notes Grace_Note_Stem Bravura
Grace_Notes Grace_Note_Stem Bravura_current
Grace_Notes Grace_Note_Stem Bravura_reference
Grace_Notes Grace_Note_Stem_with_Beams_1 Bravura
Grace_Notes Grace_Note_Stem_with_Beams_1 Bravura_current
Grace_Notes Grace_Note_Stem_with_Beams_1 Bravura_reference
Grace_Notes Grace_Note_Stem_with_Beams_2 Bravura
Grace_Notes Grace_Note_Stem_with_Beams_2 Bravura_current
Grace_Notes Grace_Note_Stem_with_Beams_2 Bravura_reference
Grace_Notes With_Articulation_and_Annotation_on_Parent_Note Bravura
Grace_Notes With_Articulation_and_Annotation_on_Parent_Note Bravura_current
Grace_Notes With_Articulation_and_Annotation_on_Parent_Note Bravura_reference
Grace_Tab_Notes Grace_Tab_Note_Simple Bravura
Grace_Tab_Notes Grace_Tab_Note_Simple Bravura_current
Grace_Tab_Notes Grace_Tab_Note_Simple Bravura_reference
Grace_Tab_Notes Grace_Tab_Note_Slurred Bravura
Grace_Tab_Notes Grace_Tab_Note_Slurred Bravura_current
Grace_Tab_Notes Grace_Tab_Note_Slurred Bravura_reference

### CODE REVIEW

**Comment by AaronDavidNewman** [comment]
This looks promising, how does it look with the fiddle: https://jsfiddle.net/xw27b9ue/

File: src/gracenotegroup.ts [hunk 1]
```diff
@@ -25,6 +25,10 @@ function L(...args: any) {
   if (GraceNoteGroup.DEBUG) log('Vex.Flow.GraceNoteGroup', args);
 }
 
+// Left padding to avoid that the group collides with the previous note.
+// The value increases the width and the x offset.
+const LEFT_PADDING = 2;
```
Comment (AaronDavidNewman) [inline src/gracenotegroup.ts:30]: You can use StaveNote.minNoteheadPadding.  It is actually defined in the font metric.

**Comment by rvilarl** [comment]
> This looks promising, how does it look with the fiddle: https://jsfiddle.net/xw27b9ue/

Much better!
https://jsfiddle.net/gL0wty82/

File: src/gracenotegroup.ts (hunk 1 above)
Comment (rvilarl) [reply src/gracenotegroup.ts]: Done, thank you!

**Comment by 0xfe** [comment]
Good fix, Rodrigo! Thanks!

**Pull request merged**